packages/block-editor/src/store/test/actions.js

```diff
@@ -180,6 +188,14 @@ describe( 'actions', () => {
 			} ];
 
 			const replaceBlockGenerator = replaceBlocks( [ 'chicken' ], blocks );
+			expect(
+				replaceBlockGenerator.next().value,
+			).toEqual( {
+				args: [],
+				selectorName: 'getSettings',
+				storeName: 'core/block-editor',
+				type: 'SELECT',
+			} );
```

Comment:
We probably don't need to retest that a `SELECT` action was yielded across this file — only once. Should we only consume the value?

```js
replaceBlockGenerator.next();
```

Reply:
I applied the suggestion.